doc/devel/saved_replies.rst

```diff
@@ -0,0 +1,95 @@
+
+.. _saved_replies:
+
+Saved replies
+=============
+
+To facilitate DRY (do not repeat yourself) communication principles, reduce potential bias and errors in
+communication, and consistently maintain respectful and welcoming communication we decided to develop and collect a set
+of reusable replies for recurring project needs.
+
+Our goal is to write welcoming and friendly communication which encourage participation and contribution in various
+ways and at different levels of technical experience, as well as clarifies what we are able, and sometimes are not able,
+to support our community with.
+
+As a maintainer, triage team member, or any other active community member, it may be helpful to store some of these in
+your GitHub account’s `saved replies`_ for reviewing issues and PRs, and supporting other community members, as they participate
+and contribute to Matplotlib.
+
+Our saved replies can be long and detailed. This is because we aim to offer support without assuming the level of
+experience, or the intent the community member we are addressing might have. There is still room for one to tweak and
+personalize the message, if based on the ongoing conversation some parts are redundant, require further information, or
+if they want to add a personal note to suggest that they will be available to offer support, for example.
+
+Our intent with providing a baseline is to make it easier to not miss the minimum amount of information, and
+consistently facilitate the respectful and welcoming interactions we aspire to maintain, not to set rigid communication
+constraints. We trust our maintainers, triage team members, and all community members who continue to abide by our
+`Code of Conduct`_, to communicate freely. It is based on their ongoing experience that we are able to provide these saved
+replies to begin with.
+
+.. _Code of Conduct: https://github.com/matplotlib/matplotlib/blob/master/CODE_OF_CONDUCT.md
+.. _saved replies: https://github.com/settings/replies/
+
+Issues
+------
+
+Coming soon 😉
+
+Pull Requests
+-------------
+
+First Pull Request Merged
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Hi-five ✋ on merging your first pull request to Matplotlib, @username! We hope you stick around and invite you to continue to take an active part in Matplotlib!
```

Comment:
Can you try to use a [block quote](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#block-quotes) to mark the actual text to be copied? 

```suggestion
Suggested reply:

    Hi-five ✋ on merging your first pull request to Matplotlib, @username! We hope you stick around and invite you to continue to take an active part in Matplotlib!
    
    ...
```

Reply:
Note, we need a block quote or code block because the reply should must be markdown not ReST. - This means also that you should change the links in the reply to markdown syntax.

Reply:
I think this should be a code-block of md in the rst ( 🤯 ) .  I just tried to use this and fixed the links my hand.